bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/TTTT_5f_NLO/TTTT_5f_NLO_customizecards.dat

```diff
@@ -0,0 +1,4 @@
+#put card customizations here (change top and higgs mass for example)
+set param_card mass 6 173.0
+set param_card yukawa 6 173.0
```

Comment:
we use 172.5 for UL, and for Run3 TTbar, we also use 172.5, so i suggest them to be consistent